beesdoo_worker_status/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    irregular_penalty = fields.Boolean(
+        string="Penalty for irregular with negative counter",
```

Comment:
You're missing the help message : "Si cette option est sélectionnée, le compteur du volant passera de 0 à -2, sauf s’il a été en alerte juste avant, auquel cas il passera de 0 à -1"

Reply:
fixed (translated in english)